tools/reckless

```diff
@@ -353,6 +353,7 @@ def _install_plugin(src: InstInfo) -> bool:
     if src.subdir is not None:
         plugin_path = Path(clone_path).joinpath(src.subdir)
     os.chdir(plugin_path)
+    assert os.getcwd() == str(plugin_path)
```

Comment:
This is likely unexpected for the user since they'll be dropped into a temporary directory if anything goes wrong. You can use the `cwd` argument to `Popen` to have it change the directory in the subprocess before exec'ing the executable, thus leaving the parent process in its current working directory unchanged.

Let's do that in a cleanup PR, since this is just a minor fix.